apps/docs/pages/guides/auth/auth-email-templates.mdx

```diff
@@ -71,6 +72,39 @@ const { data: { session }, error } = supabase.auth.verifyOtp({ token_hash, type
 
 ```
 
+## Customization
+
+Supabase Auth makes use of [Go Templates](https://pkg.go.dev/text/template). This means it is possible to conditionally render information based on template properties.You may wish to checkout this [guide by Hugo](https://gohugo.io/templates/introduction/) for a guide on the templating language.
+
+Let's demonstrate some ways of using the templating system below:
+
+### Different Email Based on Signup Domain
+
+Send users different emails based on the domain that they signed up for your service with.
+
+```
+{{ if eq .Data.Domain "http://www.example.com" }}
+<insert content for Pro Tier users>
+{{ else if eq .Data.Domain "http://www.specialsite.com" }}
+<insert content for Free Tier users>
+{{end }}
+```
+
+
+### Custom Email Content For Secure Email Change Flow
+
+Send two distinct emails: one customized for the new email account and another for the email the user is currently using.
+```
+{{ if eq .Email .SendingTo }}
+<insert content which will be received by current email>
+{{ else }}
+<insert content which will be received by new email>
+{{end}}
+
+```
```

Comment:
can we give an example of why this is useful (i.e. for the token hash validation)?